sound/soc/sof/intel/hda-ctrl.c

```diff
@@ -288,6 +292,9 @@ int hda_dsp_ctrl_init_chip(struct snd_sof_dev *sdev, bool full_reset)
 
 err:
 	hda_dsp_ctrl_misc_clock_gating(sdev, true);
+#if IS_ENABLED(CONFIG_SND_SOC_SOF_HDA)
+	snd_hdac_set_codec_wakeup(bus, false);
+#endif
```

Comment:
@kv2019i does the order between the clock gating and codec wakeup matter?

I was wondering if for extra robustness we'd want the clock gating off, with the IP having no inhibitions before detecting a wakeup?

Reply:
@plbossart No documented dependency, but this is the order in hda_intel.c so I followed that to the letter w.r.t rest of chip init actions.

Reply:
ok, that's fair, and that's what you commit message says, so we're good.